emscripten.py

```diff
@@ -776,7 +804,12 @@ def get_exported_implemented_functions(all_exported_functions, all_implemented,
     if key in all_exported_functions or export_all or (export_bindings and key.startswith('_emscripten_bind')):
       funcs.add(key)
 
-  funcs = list(funcs) + metadata['initializers']
+  funcs = list(funcs)
+  if should_group_global_initializers(metadata['initializers']):
+    funcs += ['globalCtors']
```

Comment:
It is not possible to adjust `metadata['initializers']` directly because the code flow first creates the JS side, and only then the asm.js side, meaning that we would need to destroy `metadata['initializers']` already on JS side creation, but still would need to remember what the values of the initializers were in order to create the asm.js side grouped initializer.

Updated the code to avoid duplicate ifs in a slightly different way to get the same effect.